x/ecocredit/client/tx.go

```diff
@@ -297,8 +298,51 @@ Example JSON:
 // TxSendCmd returns a transaction command that sends credits from one account
 // to another.
 func TxSendCmd() *cobra.Command {
+	var retireTo string = ""
 	cmd := &cobra.Command{
-		Use:   "send [recipient] [credits]",
+		Use:   "send [amount] [batch_denom] [recipient]",
+		Short: "Sends credits from the transaction author (--from) to the recipient",
+		Long: `Sends credits from the transaction author (--from) to the recipient.
+
+Parameters:
+  amount: amount to send
+  batch_denom: batch denomination
+  recipient: recipient address`,
+		Args: cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := sdkclient.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			tradableAmount := args[0]
+			retiredAmount := "0"
+			if len(retireTo) > 0 {
+				tradableAmount = "0"
+				retiredAmount = args[0]
+			}
+			credit := core.MsgSend_SendCredits{
+				TradableAmount:         tradableAmount,
+				BatchDenom:             args[1],
+				RetiredAmount:          retiredAmount,
+				RetirementJurisdiction: retireTo,
+			}
+			msg := core.MsgSend{
+				Sender:    clientCtx.GetFromAddress().String(),
+				Recipient: args[2],
+				Credits:   []*core.MsgSend_SendCredits{&credit},
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), &msg)
+		},
+	}
+	cmd.Flags().StringVar(&retireTo, "retire-to", "", "Jurisdiction to retire the credits to. If empty, credits are not retired. (default empty)")
+	return txFlags(cmd)
+}
+
+// TxSendBulkCmd returns a transaction command that sends credits from one account
+// to another.
```

Comment:
```suggestion
// TxSendBulkCmd returns a transaction command that can send credits from multiple batches from one account
// to another.
```

Reply:
not sure if this is the best doc, maybe @ryanchristo can suggest something cleaner given these udpates